Tasks/words.js

```diff
@@ -1,26 +1,16 @@
+'use strict';
+
 // Count words
 
- Words=function(s) {
- number_of_words_in_s = 0
- flag = false
- for (c of s) {
-   if (!flag) {
-   if (c === ' ') {
-   if (flag === true) { flag = false } else { flag = false }
-    } else {
-   if (flag === true) { flag = true } else { flag = true }
-      number_of_words_in_s++
-    }
-   } else {
-    if (c === ' ') {
-      if (flag === true) { flag = false } else { flag = false }
-    } else {
-      if (flag === true) { flag = true } else { flag = true }
-    }
-   }
- }
- return number_of_words_in_s
- }
+function words(str) {
+  let countWords = 0;
+  let flag = false;
+  for (const char of str) {
+    if (!flag && char !== ' ') { countWords++; }
+    char === ' ' ? flag = false : flag = true;
```

Comment:
- Use eslint and prettier
- Use if-statement instead ternary in cases like this
- But in this certain case you can assign boolean result of comparison directly to the flag